ios/Runner/Info.plist

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>BGTaskSchedulerPermittedIdentifiers</key>
+	<array>
+		<string>com.example.mosquito_alert_app.scheduleDailyTasks</string>
+    	<string>com.example.mosquito_alert_app.trackingTask</string>
```

Comment:
I guess both value must be equal to the ones declared in AppDelegate.swift, so then:
```
    <string>trackingTask</string>
    <string>scheduleDailyTasks</string>
```